code/processes/segmentedtickerplant.q

```diff
@@ -83,5 +83,38 @@ init:{
   .stplg.init[string .proc.procname];
  };
 
-// Have the init function called from torq.q
+// Loads the striping.json config file and logs any problems
+configload:{[scpath]
+     @[jsonchecks; 
+       .stpps.stripeconfig:@[{.j.k read1 x};scpath;{.lg.o[`configload;"Failed to load in striping.json file: ",x]}];
+       {.lg.o[`configload;"Datastriping is not enabled due to problems with striping.json"]}]
+     };
+
+// Checks if each subscriptiondefault is set for each segment and errors if not defined
+jsonchecks:{[.stpps.stripeconfig]
```

Comment:
this looks weird - is a variable being passed in here, or are we referrencing a global variable in the `.stpps` namespace?

if we're referrencing a global variable, just make this

```suggestion
jsonchecks:{[]
```

and pass a null or something when calling it above